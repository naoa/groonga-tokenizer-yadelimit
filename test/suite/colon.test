register tokenizers/yadelimit

tokenize TokenYaDelimitColon "a iu. eo: kaki@kuke.com"
