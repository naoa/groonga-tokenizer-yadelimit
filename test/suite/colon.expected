register tokenizers/yadelimit
[[0,0.0,0.0],true]
tokenize TokenYaDelimitColon "a iu. eo: kaki@kuke.com"
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "a iu. eo",
      "position": 0
    },
    {
      "value": " kaki@kuke.com",
      "position": 1
    }
  ]
]
